baselines/polars/tpch/Makefile

```diff
@@ -0,0 +1,56 @@
+SHELL=/bin/bash
+PYTHON=.venv/bin/python
+
+.venv:
+	@python -m venv .venv
+	@.venv/bin/pip install -U pip
+	@.venv/bin/pip install --no-cache-dir -r requirements.txt
+
+clean-tpch-dbgen:
+	$(MAKE) -C tpch-dbgen clean
+
+clean-venv:
+	rm -r .venv
+
+clean-tables:
+	rm -r tables_scale_*
+
+clean: clean-tpch-dbgen clean-venv
+
+tables_scale_1: .venv
+	$(MAKE) -C tpch-dbgen all
+	cd tpch-dbgen && ./dbgen -vf -s 1 && cd ..
+	mkdir -p "tables_scale_1"
+	mv tpch-dbgen/*.tbl tables_scale_1/
+	.venv/bin/python prepare_files.py 1
+
+tables_scale_10: .venv
+	$(MAKE) -C tpch-dbgen all
+	cd tpch-dbgen && ./dbgen -vf -s 10 && cd ..
+	mkdir -p "tables_scale_10"
+	mv tpch-dbgen/*.tbl tables_scale_10/
+	.venv/bin/python prepare_files.py 10
+
+run_polars: .venv
+	.venv/bin/python -m polars_queries.executor
+
+run_pandas: .venv
```

Comment:
Since we only have polars, we don't need these other `run_` targets.

Reply:
I delete all extra `run_` targets except for `run_polars`